pymc/distributions/continuous.py

```diff
@@ -1235,6 +1235,16 @@ def logcdf(value, alpha, beta):
             msg="alpha > 0, beta > 0",
         )
 
+    def icdf(value, alpha, beta):
+        res = betaincinv(alpha, beta, value)
+        res = check_icdf_value(res, value)
+        return check_parameters(
```

Comment:
Should use https://github.com/pymc-devs/pymc/blob/main/pymc%2Fdistributions%2Fdist_math.py#L76

And in the other functions 

This is because the default `check_parameters` was developed for logp in mind and pymc replaces it by -inf sometimes, which doesn't make sense for icdf expressions

Reply:
Ops, my bad! Actually I just realized that some of the other icdf functions I have implemented were using `check_parameters` instead of `check_icdf_parameters`. I've changed them in the commit https://github.com/pymc-devs/pymc/pull/6845/commits/be24e627a698659037db8cb8affb9c82ad24d2ee, but if you think those are out of topic, I can revert it and create a new PR only for these small changes.

Reply:
It's fine to do it here. Thanks for the fix

Reply:
Can you just squash the commits related to the new functionality into one and have the fix as a separate commit?

Reply:
I will try that! It will be an interesting experience.

Reply:
After failing quite a bit, it is done!
Icdf implementations + tests in https://github.com/pymc-devs/pymc/pull/6845/commits/babc47923fd8272014be2c2f948ad917cfc60609
Fixing `check_parameters` --> `check_icdf_parameters` in https://github.com/pymc-devs/pymc/pull/6845/commits/c54cd1d05cfe9fabd913bcf1479dc6c730cd53b6

Reply:
Thanks for guiding me through this. I always used the "update branch" in the github UI at the top of the repository, whenever I start working or before the revision. 
The previous "merge main" was kind of a hassle when I tried to rebase to squash commits. It was a good learning lesson, and from now on I will shift to pull rebase.

Reply:
I'm still learning myself. The recommended source, if you want to learn in less randomly, is usually https://git-scm.com/book/en/v2

Some chapters are more useful than others if you already know you are working specifically with git + github

Reply:
Nice!! Thank you for the tip! I definitely want to level up in git skills.